web_widget_plotly_chart/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Web Widget Plotly",
+    "summary": """Allow to draw plotly charts.""",
+    "author": "LevelPrime srl, " "Odoo Community Association (OCA)",
```

Comment:
Extra quotes are unnecessary here.

Reply:
@tarteo Do you me on line 3 to change the triple quote and on line 4 to make one string?

Reply:
![image](https://user-images.githubusercontent.com/10028499/145389013-f95f3526-2cef-412d-b087-b76fbba2f446.png)
These

Reply:
@tarteo Done